pom.xml

```diff
@@ -109,27 +109,40 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
-      <version>1.7.18</version>
+      <version>1.7.30</version>
     </dependency>
-
     <!-- we use log4j as our logging implementation -->
+    <!--
+      The Log4j 2 SLF4J Binding allows applications coded to the SLF4J API to use Log4j 2 as the implementation.
+
+      Due to a break in compatibility in the SLF4J binding, as of release 2.11.1 two SLF4J to Log4j Adapters are provided.
+
+      log4j-slf4j-impl should be used with SLF4J 1.7.x releases or older.
+      log4j-slf4j18-impl should be used with SLF4J 1.8.x releases or newer.
+      Applications that take advantage of the Java Module System should use SLF4J 1.8.x and log4j-slf4j18-impl.
+
+      The Log4j 2 SLF4J Binding has a dependency on the Log4j 2 API as well as the SLF4J API
+    -->
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
-      <version>1.7.13</version>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
+      <version>2.13.3</version>
     </dependency>
+    <!-- Log4j implementation of Log4j2 -->
     <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
-      <version>1.2.17</version>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-bom</artifactId>
+      <version>2.12.1</version>
+      <scope>import</scope>
+      <type>pom</type>
     </dependency>
 
     <!-- include slf4j bridges to proxy all logging from library deps to
       our logging impl -->
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>jcl-over-slf4j</artifactId>
-      <version>1.7.18</version>
+      <version>1.7.30</version>
```

Comment:
If this is always going to move at the same pace as slf4j-api, you might consider pulling this out into a variable so it only needs to be changed in one place.

Reply:
Good point, will fix.